### PR TITLE
Fixes #1173: Change Telescope theme to a light theme

### LINKED ISSUE #1173: Change Telescope Theme to a light theme
**What would you like to be added**:

Change the color of the components to match this new light design, which has better contrast.
image
image
image
image

**Why would you like this to be added**:

The current theme has contrast and user experience issues, #1055 and #1039, so the new light theme would improve the Telescope UI and provide for a better reader experience.

### PR DESCRIPTION
**Pull request by hyunjiLeeTech**
## Issue This PR Addresses

#1173. Change Telescope Theme to a light theme

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

Homepage
image

About page
image

My feed page
image

404 page
image

While I'am working on lighten the 404 page, I've found the homepage, about page, and my feed page also got lighter theme following my change. I've changed background and primary colour on 'theme.js' file. Please have a look at this colour change is whether what you or not and let me know if you want another colour theme.

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Review by agarcia-caicedo — CHANGES_REQUESTED** [review]
Hi, to turn convert the 404 page to light mode, you shouldn't change the theme.js file, that changes everything that uses that file. Hence why all the other pages got changed. We already decided on the colours in pr #1174

**Review by agarcia-caicedo — CHANGES_REQUESTED** [review]
In the error.jsx file, can you change the color of the font for the "let's go back" button, so that it uses the dark grey instead of the off-white? The contrast isn't great with the white

**Comment by hyunjiLeeTech** [comment]
Hi, sorry about the changes of the background and default colour. I've changed background/default colour to original colour you've decided (#e5e5e5) and changed the "let's go back" button colour to "#d3d3d3". Please let me know if you need another change or want another colour. Thank you.

File: src/frontend/src/pages/error.jsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ const useStyles = makeStyles((theme) => ({
   fab: {
     position: 'relative',
     fontSize: '1.5rem',
-    color: 'white',
+    color: '#D3D3D3',
```
Comment (agarcia-caicedo) [inline src/frontend/src/pages/error.jsx:75]: Sorry, I should have been clearer. This "color: theme.palette.text.default" is what I meant.

**Review by agarcia-caicedo — CHANGES_REQUESTED** [review]

**Comment by hyunjiLeeTech** [comment]
I've change the font colour to "theme.pallette.text.default". This is how it looks like now :). Please let me know if you want something more about it. Thanks.
image

**Review by agarcia-caicedo — APPROVED** [review]
Thanks for the contribution!

**Review by humphd — APPROVED** [review]

**Pull request merged**